packages/evolution-frontend/src/components/admin/validations/InterviewMap.tsx

```diff
@@ -335,10 +449,70 @@ const InterviewMap: React.FunctionComponent<InterviewMapProps> = (props: Intervi
                 [Math.max(...lngs), Math.max(...lats)]
             ];
 
-            (map as any).fitBounds(bounds, { padding: 50 });
+            map.fitBounds(bounds, { padding: 50, animate: false, maxZoom: 17 });
         }
     };
 
+    /**
+     * Fit map bounds to show the active trip's origin and destination
+     */
+    const fitBoundsToActiveTrip = () => {
+        const map = mapRef.current?.getMap();
+        if (!map || !props.activeTripUuid) return;
+
+        // Find the active trip
+        const activeTrip = props.trips.features.find(
+            (feature) => feature.properties?.tripUuid === props.activeTripUuid
+        );
+
+        if (!activeTrip || !activeTrip.geometry.coordinates.length) return;
+
+        const coordinates = activeTrip.geometry.coordinates;
+
+        // Get origin (first coordinate) and destination (last coordinate)
+        const origin = coordinates[0] as [number, number];
+        const destination = coordinates[coordinates.length - 1] as [number, number];
+
+        // Create bounds from origin and destination
+        const lngs = [origin[0], destination[0]];
+        const lats = [origin[1], destination[1]];
+
+        const bounds: [[number, number], [number, number]] = [
+            [Math.min(...lngs), Math.min(...lats)],
+            [Math.max(...lngs), Math.max(...lats)]
+        ];
+
+        // Add some padding and animate to the bounds
+        map.fitBounds(bounds, {
+            padding: 100, // More padding for better view of origin/destination
+            animate: true,
+            duration: 1000, // Smooth animation
+            maxZoom: 17 // Limit zoom to prevent blurry tiles
+        });
+    };
+
+    /**
+     * Zoom to active visited place at zoom level 16
+     */
+    const zoomToActivePlace = () => {
+        const map = mapRef.current?.getMap();
+        if (!map || !props.activePlacePath) return;
+
+        // Find the active place
+        const activePlace = props.places.features.find((feature) => feature.properties?.path === props.activePlacePath);
+
+        if (!activePlace || !activePlace.geometry.coordinates) return;
+
+        const coordinates = activePlace.geometry.coordinates as [number, number];
+
+        // Smoothly animate to the place with zoom level 16
```

Comment:
says smootly zoom to level 16, but it's 14. Same in function doc. Either drop the level from the documentation, or make it a constant and make sure both match